06_modules_lab/01.py

```diff
@@ -1,4 +1,6 @@
-""" Write a program that takes a count
-from sys.argv import and prints "Hello Python"
-count times
-"""
+import sys
+
+
+for i in range(int(sys.argv[1])):
+    print "Hello World"
+

```

Comment:
👍 

Bonus: What if no argument was passed or it was not a number? Can you show a nicer error message to the user?